Ecommerce/index.css

```diff
@@ -2,4 +2,9 @@
 img {

```

Comment:
There should not be a closing curly brace without a matching opening curly brace.